lib/transporters.js

```diff
@@ -39,8 +39,11 @@ DefaultTransporter.prototype.USER_AGENT =
 DefaultTransporter.prototype.configure = function(opts) {
   // set transporter user agent
   opts.headers = opts.headers || {};
-  opts.headers['User-Agent'] = opts.headers['User-Agent'] ?
-    opts.headers['User-Agent'] + ' ' + this.USER_AGENT : this.USER_AGENT;
+  if (!opts.headers['User-Agent']) {
+    opts.headers['User-Agent'] = this.USER_AGENT;
+  } else if (!opts.headers['User-Agent'].endsWith(this.USER_AGENT)) {
```

Comment:
I think we should find another way to check this, since this is only compatible with Node 4 and up.